.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,25 @@
+# Citizen Code of Conduct
+
+Our goal was to improve Prodigy's security, not ruining the game for profit. We won't sell our hacks, or put ads.
+
+Our hacks will **always** be open-source, and free to use.
+
+---
+
+You may ***never***:
+
+- sell our hacks in any way shape or form
+- hide our hacks behind a paywall/subscribewall of any kind
+  - This includes adfly 😐
+- use our hacks to leak private information
+- use our hacks to steal user login information or tokens
+- claim ownership of our hacks (Just stop bragging about it 😠)
+- modify it slightly just to add your own ads 🤨
+
+If you're gonna talk about our hacks (on YouTube or other social media platforms), please please pretty please [credit us properly](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/blob/master/YOUTUBE_ATTRIBUTION.md).
+
+It hurts our souls when skids take credit for our hacks and don't even link us.
```

Comment:
lmao okay

Reply:
> why do you have 3 alts

Wdymmm....... it's just many people like my pr....


whatever